Package.swift

```diff
@@ -24,6 +24,7 @@ import PackageDescription
 
 let package = Package(
   name: "SwiftPhoenixClient",
+  platforms: [.iOS(.v10), .macOS(.v10_12), .tvOS(.v10), .watchOS(.v2)],
```

Comment:
current podspec has iOS at 9.0. Could you please set `.iOS(.v9)`.

Reply:
my mistake, on it